cmd/osbuild-composer/main.go

```diff
@@ -66,6 +82,9 @@ func main() {
 				KeyTab    string `toml:"keytab"`
 			} `toml:"kerberos,omitempty"`
 		} `toml:"koji"`
+		RemoteWorker *struct {
+			DomainAllowlist []string `toml:"domain_allowlist"`
```

Comment:
Just call this `domains`?

Reply:
`allowedDomains`? I like more context...